Workflows/ExperimentAcquisitionVR.bonsai

```diff
@@ -1,1267 +1,1730 @@
+                  <Expression xsi:type="rx:BehaviorSubject" TypeArguments="sys:Tuple(sys:String,p2:NetMQMessage)">
+                    <rx:Name>OmniceptGlia</rx:Name>
+                  </Expression>
+                  <Expression xsi:type="rx:GroupBy">
+                    <rx:KeySelector>Item1</rx:KeySelector>
+                    <rx:ElementSelector>Item2</rx:ElementSelector>
                   </Expression>
-                  <Expression xsi:type="rx:Sink">
+                  <Expression xsi:type="IncludeWorkflow" Path="Extensions\ZeroMQLogging.bonsai">
+                    <LogName>Glia</LogName>
+                    <StartIndex>0</StartIndex>
+                    <ExpectedIndices>3</ExpectedIndices>
+                  </Expression>
+                  <Expression xsi:type="rx:BehaviorSubject" TypeArguments="p3:IplImage">
+                    <rx:Name>VRImage</rx:Name>
+                  </Expression>
+                  <Expression xsi:type="GroupWorkflow">
+                    <Name>LogVideo</Name>
                     <Workflow>
                       <Nodes>
                         <Expression xsi:type="WorkflowInput">
                           <Name>Source1</Name>
                         </Expression>
-                        <Expression xsi:type="GroupWorkflow">
-                          <Name>Harp Logging</Name>
+                        <Expression xsi:type="SubscribeSubject">
+                          <Name>LoggingPath</Name>
+                        </Expression>
+                        <Expression xsi:type="ExternalizedMapping">
+                          <Property Name="Value" DisplayName="LogName" />
+                        </Expression>
+                        <Expression xsi:type="Combinator">
+                          <Combinator xsi:type="StringProperty">
+                            <Value>VRImage.avi</Value>
+                          </Combinator>
+                        </Expression>
+                        <Expression xsi:type="Combinator">
+                          <Combinator xsi:type="rx:Zip" />
+                        </Expression>
+                        <Expression xsi:type="Format">
+                          <Format>{0}\{1}</Format>
+                          <Selector>Item1,Item2</Selector>
+                        </Expression>
+                        <Expression xsi:type="PropertyMapping">
+                          <PropertyMappings>
+                            <Property Name="FileName" />
+                          </PropertyMappings>
+                        </Expression>
+                        <Expression xsi:type="ExternalizedMapping">
+                          <Property Name="FrameRate" />
+                        </Expression>
+                        <Expression xsi:type="Combinator">
+                          <Combinator xsi:type="cv:VideoWriter">
+                            <cv:FileName>C:\IndoorData\NGR_LAB_sub-01_2024-07-31T185605Z\\VRImage.avi</cv:FileName>
```

Comment:
Remove absolute path

Reply:
Fixed in b3c8402bc14396a948360dacd8b5c03789151b28